webrtc.html

```diff
@@ -3855,6 +3856,10 @@
               <dd>A unique identifier that allows ICE to correlate candidates
               that appear on multiple
               <code><a>RTCIceTransport</a></code>s.</dd>
+              <dt><dfn><code>component</code></dfn> of type <span class=
+              "idlAttrType"><a>byte</a></span>, readonly, nullable</dt>
+              <dd>The assigned network component id of the candidate
+              (1 for <code>rtp</code>, 2 for <code>rtcp</code>).</dd>
```

Comment:
nit: Why are "rtp" and "rtcp" in "code" blocks?

Also, this could reference ICEBis section 4.1.1.1 which defines the term "component id".